el/pom.xml

```diff
@@ -46,6 +46,16 @@
             <groupId>jakarta.el</groupId>
             <artifactId>jakarta.el-api</artifactId>
         </dependency>
+       <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
```

Comment:
Would it be possible to avoid dependency on SLF4J and use System.Logger instead?
See https://www.slf4j.org/manual.html#jep264

Reply:
Done.

Reply:
Why?
Better to have something without real implementations and user can choose whatever implementations they prefer.
If you prefer `System.Logger` fine you add this slf4j dependency at runtime, if another user prefere log4j or logback fine they just add the correct slf4j binding at runtime.
Here you force user with an implementation they may not want.
slf4j have the advantage of being implementation neutral and users have the freedom to choose the one they prefer.

Reply:
SLF4J is NOT neutral, it is an additional dependency which is not necessary these days. System.Logger is the neutral API, which is inside Java since Java 9. SLF4J2 can bind to it too just like other logging frameworks.
See also https://openjdk.org/jeps/264 - the SLF4J is even explicitly mentioned.

Reply:
oh right thanks!
 I didn't look at the other way, another cascade :) `System.Logger` -> `slf4j api` -> `final impl` 
that's a bit of layers but that's fine.